packages/client/src/runtime/strictEnum.ts

```diff
@@ -1,5 +1,16 @@
 export const strictEnumNames = ['TransactionIsolationLevel']
 
+/**
+ * List of properties that won't throw exception on access and return undefined instead
+ */
+const allowList = new Set([
+  'toJSON', // used by JSON.stringify
+  'asymmetricMatch', // used by Jest
```

Comment:
Bit odd we have to change the src code for testing.

Reply:
Jest will try to get a property every time you do `toEqual(StrictEnum)` or `.toBe(StrictEnum)`. Not only in our tests, but in users tests as well.
Normally I'd agree with you, but at the same time, i think Jest is popular enough to warrant a fix on our side in this case.